src/commands/update.rs

```diff
@@ -21,11 +21,17 @@ pub fn make_app<'a, 'b: 'a>(app: App<'a, 'b>) -> App<'a, 'b> {
 }
 
 pub fn execute<'a>(matches: &ArgMatches<'a>) -> Result<(), Error> {
+    // Disable update check in case of errors
+    env::set_var("SENTRY_DISABLE_UPDATE_CHECK", "true");
+
+    // Aborts with an error if this installation is not updatable.
+    assert_updatable()?;
```

Comment:
I think it makes sense to check this before hitting the registry to save some requests.

Reply:
@jan-auer We also want to avoid re-running the update when we have any kind of errors:

https://github.com/getsentry/sentry-cli/blob/31aacf105125f7797fb29ac5c468e6da7d7ec699/src/commands/mod.rs#L255